extensions/cpp/package.json

```diff
@@ -71,8 +82,35 @@
       {
         "scopeName": "source.c.platform",
         "path": "./syntaxes/platform.tmLanguage.json"
+      },
+      {
+        "language": "cuda-cpp",
```

Comment:
With the new grammar, we should also have new colorizer tests. Colorizer tests are very easy to add. Simply add a file with the `.cu` or `.cuh` extension to \extensions\vscode-colorize-tests\test\colorize-fixtures\. Then, from your cloned VS Code repo, run .\scripts\test-integration.bat (or .sh). This will generate the colorizer test result, which lets us check for possible breaking changes when the grammar is updated.

Reply:
Test file looks good 👍

Reply:
I created a `.cu` file under colorize-fixtures and then ran ./scripts/test-integration.sh, I get `"ERROR: There are no scenarios; must have at least one."` Is there another step?